### ISSUE TITLE
Measure in Model does not show any further settings.

### ISSUE THREAD

**Issue by maryam-fatima** [makehumancommunity/mpfb2#167]
**Describe the bug**
The **measure** option in **Model** section does not show anything.

**Screenshots / logs**
<img width="1071" alt="Screenshot 2024-01-04 at 2 05 02 PM" src="https://github.com/makehumancommunity/mpfb2/assets/62232940/1c3755c5-ef75-4fda-998e-6d9a49e57431">

**Platform and versions**
 - OS: Mac
 - Version of Blender: 3.6.2
 - Version of MPFB: v2.0-alpha3

**Comment by joepal1976** [comment]
Yes, that's a glitch in the UI. The measure targets are available, they're just sorted under what area of the body they belong to. For example waist and hip circumference targets are under the torso category:

![image](https://github.com/makehumancommunity/mpfb2/assets/10105480/09d58f73-5a75-4198-b957-1a2a58af6181)

The category should be removed to avoid confusion though, so I'll leave this issue open.

**Comment by maryam-fatima** [comment]
okay, so there is no way to use the measurements in metric/imperial units in MPFB like the option we have in Measure tab under Modeling in MakeHuman GUI?

**Comment by joepal1976** [comment]
No, that has not been implemented yet. The modeling sliders are available, but they are in the 0.0 to 1.0 range rather than  projected to real world measurements.

**Comment by LibrEars** [comment]
Maybe it would be beneficial if the sliders are shown in both (measure tab and inside the category)
